src/preprocessing/passes/quantifier_macro_elimination.h

```diff
@@ -0,0 +1,57 @@
+#include "cvc4_private.h"
+
+#ifndef CVC4__PREPROCESSING_PASSES_FOO_H
+#define CVC4__PREPROCESSING_PASSES_FOO_H
+
+#include <unordered_map>
+#include <unordered_set>
+#include <vector>
+
+#include "context/context.h"
+#include "expr/node.h"
+#include "preprocessing/preprocessing_pass.h"
+#include "preprocessing/preprocessing_pass_context.h"
+#include "theory/logic_info.h"
+#include "theory/substitutions.h"
+
+namespace CVC4 {
+namespace preprocessing {
+namespace passes {
+using namespace std;
+
+// TODO: are these already defined somewhere else?
```

Comment:
No.

I generally don't like defining these since it makes the code harder to follow in my opinion (I tend to be paranoid and don't want to have to follow definitions of whether the helper function is defined as expected).